tests/test_api.py

```diff
@@ -34,7 +33,7 @@ def setup_propagate_exceptions(propagate_exceptions):
 
 
 def check_unpack(expected, value):
-    assert_equal(expected, value)
+    assert expected == value
 
 
 def test_unpack():
```

Comment:
`PytestCollectionWarning: yield tests were removed in pytest 4.0 - test_unpack will be ignored`